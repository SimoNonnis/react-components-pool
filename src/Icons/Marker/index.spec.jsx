/* eslint-env jest */
import React from 'react';

import Marker from './';

describe('Test <Marker /> icon', () => {
  it('should render Marker icon correctly with default props', () => {
    const wrapper = shallow(<Marker />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render Marker icon correctly with props', () => {
    const wrapper = shallow(<Marker fill="#a4a1a1" className="pof" height={30} width={30} />);
    expect(wrapper).toMatchSnapshot();
  });
});
