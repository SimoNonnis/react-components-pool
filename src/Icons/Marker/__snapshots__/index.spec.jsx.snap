// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Marker /> icon should render Marker icon correctly with default props 1`] = `
<svg
  className="markerIcon"
  height={40}
  viewBox="0 0 100 100"
  width={40}
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    Marker Icon
  </title>
  <path
    className="markerIconPath"
    d="M279-379.17a43.58,43.58,0,0,1-2.63-3.81,9.21,9.21,0,0,1-1.6-3.86,4.23,4.23,0,0,1,2.11-3.66,4.23,4.23,0,0,1,4.23,0,4.23,4.23,0,0,1,2.11,3.66,9.22,9.22,0,0,1-1.6,3.86A43.59,43.59,0,0,1,279-379.17Zm-2.74-2.22c1.17,1.78,2.36,3.26,2.38,3.27a.47.47,0,0,0,.36.17.47.47,0,0,0,.36-.17s1.21-1.5,2.37-3.27l.18-.27h1.44l2.37,4.66H272.23l2.37-4.66H276Zm10.65,4.64L284-382.35a.47.47,0,0,0-.42-.26h-1.14a9.49,9.49,0,0,0,1.66-4.23A5.17,5.17,0,0,0,279-392a5.17,5.17,0,0,0-5.17,5.17,9.51,9.51,0,0,0,1.66,4.23h-1.14a.47.47,0,0,0-.42.26l-2.85,5.59a.47.47,0,0,0,0,.46.47.47,0,0,0,.4.22h15a.47.47,0,0,0,.4-.22A.47.47,0,0,0,286.87-376.75Z"
    fill="currentColor"
  />
  <path
    className="markerIconPath"
    d="M236-392.17a43.58,43.58,0,0,1-2.63-3.81,9.21,9.21,0,0,1-1.6-3.86,4.23,4.23,0,0,1,2.11-3.66,4.23,4.23,0,0,1,4.23,0,4.23,4.23,0,0,1,2.11,3.66,9.22,9.22,0,0,1-1.6,3.86A43.59,43.59,0,0,1,236-392.17Zm-2.74-2.22c1.17,1.78,2.36,3.26,2.38,3.27a.47.47,0,0,0,.36.17.47.47,0,0,0,.36-.17s1.21-1.5,2.37-3.27l.18-.27h1.44l2.37,4.66H229.23l2.37-4.66H233Zm10.65,4.64L241-395.35a.47.47,0,0,0-.42-.26h-1.14a9.49,9.49,0,0,0,1.66-4.23A5.17,5.17,0,0,0,236-405a5.17,5.17,0,0,0-5.17,5.17,9.51,9.51,0,0,0,1.66,4.23h-1.14a.47.47,0,0,0-.42.26l-2.85,5.59a.47.47,0,0,0,0,.46.47.47,0,0,0,.4.22h15a.47.47,0,0,0,.4-.22A.47.47,0,0,0,243.87-389.75Z"
    fill="currentColor"
  />
  <path
    className="markerIconPath"
    d="M194.62-392.17A43.58,43.58,0,0,1,192-396a9.21,9.21,0,0,1-1.6-3.86,4.23,4.23,0,0,1,2.11-3.66,4.23,4.23,0,0,1,4.23,0,4.23,4.23,0,0,1,2.11,3.66,9.22,9.22,0,0,1-1.6,3.86A43.59,43.59,0,0,1,194.62-392.17Zm-2.74-2.22c1.17,1.78,2.36,3.26,2.38,3.27a.47.47,0,0,0,.36.17.47.47,0,0,0,.36-.17s1.21-1.5,2.37-3.27l.18-.27H199l2.37,4.66H187.89l2.37-4.66h1.44Zm10.65,4.64-2.84-5.59a.47.47,0,0,0-.42-.26h-1.14a9.49,9.49,0,0,0,1.66-4.23,5.17,5.17,0,0,0-5.17-5.17,5.17,5.17,0,0,0-5.17,5.17,9.51,9.51,0,0,0,1.66,4.23H190a.47.47,0,0,0-.42.26l-2.85,5.59a.47.47,0,0,0,0,.46.47.47,0,0,0,.4.22h15a.47.47,0,0,0,.4-.22A.47.47,0,0,0,202.52-389.75Z"
    fill="currentColor"
  />
  <path
    className="markerIconPath"
    d="M237.19-399.79A1.19,1.19,0,0,1,236-398.6a1.19,1.19,0,0,1-1.19-1.19A1.19,1.19,0,0,1,236-401a1.19,1.19,0,0,1,1.19,1.19ZM236-401.91a2.13,2.13,0,0,0-2,1.31,2.13,2.13,0,0,0,.46,2.32,2.13,2.13,0,0,0,2.32.46,2.13,2.13,0,0,0,1.31-2A2.13,2.13,0,0,0,236-401.91Z"
    fill="currentColor"
  />
  <path
    className="markerIconPath"
    d="M254.66-398.26a43.58,43.58,0,0,1-2.63-3.81,9.21,9.21,0,0,1-1.6-3.86,4.23,4.23,0,0,1,2.11-3.66,4.23,4.23,0,0,1,4.23,0,4.23,4.23,0,0,1,2.11,3.66,9.22,9.22,0,0,1-1.6,3.86A43.59,43.59,0,0,1,254.66-398.26Zm-2.74-2.22c1.17,1.78,2.36,3.26,2.38,3.27a.47.47,0,0,0,.36.17.47.47,0,0,0,.36-.17s1.21-1.5,2.37-3.27l.18-.27H259l2.37,4.66H247.93l2.37-4.66h1.44Zm10.65,4.64-2.84-5.59a.47.47,0,0,0-.42-.26h-1.14a9.49,9.49,0,0,0,1.66-4.23,5.17,5.17,0,0,0-5.17-5.17,5.17,5.17,0,0,0-5.17,5.17,9.51,9.51,0,0,0,1.66,4.23H250a.47.47,0,0,0-.42.26l-2.85,5.59a.47.47,0,0,0,0,.46.47.47,0,0,0,.4.22h15a.47.47,0,0,0,.4-.22A.47.47,0,0,0,262.56-395.84Z"
    fill="currentColor"
  />
  <path
    className="markerIconPath"
    d="M49.21,7.74A29.37,29.37,0,0,0,19.87,37.08c0,17.91,21.49,44.22,29.34,53.24,7.85-9,29.35-35.35,29.35-53.23A29.38,29.38,0,0,0,49.21,7.74ZM59.15,37.5a9.94,9.94,0,1,1-9.94-9.94A9.94,9.94,0,0,1,59.15,37.5Z"
    fill="currentColor"
  />
</svg>
`;

exports[`Test <Marker /> icon should render Marker icon correctly with props 1`] = `
<svg
  className="pof"
  height={30}
  viewBox="0 0 100 100"
  width={30}
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    Marker Icon
  </title>
  <path
    className="markerIconPath"
    d="M279-379.17a43.58,43.58,0,0,1-2.63-3.81,9.21,9.21,0,0,1-1.6-3.86,4.23,4.23,0,0,1,2.11-3.66,4.23,4.23,0,0,1,4.23,0,4.23,4.23,0,0,1,2.11,3.66,9.22,9.22,0,0,1-1.6,3.86A43.59,43.59,0,0,1,279-379.17Zm-2.74-2.22c1.17,1.78,2.36,3.26,2.38,3.27a.47.47,0,0,0,.36.17.47.47,0,0,0,.36-.17s1.21-1.5,2.37-3.27l.18-.27h1.44l2.37,4.66H272.23l2.37-4.66H276Zm10.65,4.64L284-382.35a.47.47,0,0,0-.42-.26h-1.14a9.49,9.49,0,0,0,1.66-4.23A5.17,5.17,0,0,0,279-392a5.17,5.17,0,0,0-5.17,5.17,9.51,9.51,0,0,0,1.66,4.23h-1.14a.47.47,0,0,0-.42.26l-2.85,5.59a.47.47,0,0,0,0,.46.47.47,0,0,0,.4.22h15a.47.47,0,0,0,.4-.22A.47.47,0,0,0,286.87-376.75Z"
    fill="#a4a1a1"
  />
  <path
    className="markerIconPath"
    d="M236-392.17a43.58,43.58,0,0,1-2.63-3.81,9.21,9.21,0,0,1-1.6-3.86,4.23,4.23,0,0,1,2.11-3.66,4.23,4.23,0,0,1,4.23,0,4.23,4.23,0,0,1,2.11,3.66,9.22,9.22,0,0,1-1.6,3.86A43.59,43.59,0,0,1,236-392.17Zm-2.74-2.22c1.17,1.78,2.36,3.26,2.38,3.27a.47.47,0,0,0,.36.17.47.47,0,0,0,.36-.17s1.21-1.5,2.37-3.27l.18-.27h1.44l2.37,4.66H229.23l2.37-4.66H233Zm10.65,4.64L241-395.35a.47.47,0,0,0-.42-.26h-1.14a9.49,9.49,0,0,0,1.66-4.23A5.17,5.17,0,0,0,236-405a5.17,5.17,0,0,0-5.17,5.17,9.51,9.51,0,0,0,1.66,4.23h-1.14a.47.47,0,0,0-.42.26l-2.85,5.59a.47.47,0,0,0,0,.46.47.47,0,0,0,.4.22h15a.47.47,0,0,0,.4-.22A.47.47,0,0,0,243.87-389.75Z"
    fill="#a4a1a1"
  />
  <path
    className="markerIconPath"
    d="M194.62-392.17A43.58,43.58,0,0,1,192-396a9.21,9.21,0,0,1-1.6-3.86,4.23,4.23,0,0,1,2.11-3.66,4.23,4.23,0,0,1,4.23,0,4.23,4.23,0,0,1,2.11,3.66,9.22,9.22,0,0,1-1.6,3.86A43.59,43.59,0,0,1,194.62-392.17Zm-2.74-2.22c1.17,1.78,2.36,3.26,2.38,3.27a.47.47,0,0,0,.36.17.47.47,0,0,0,.36-.17s1.21-1.5,2.37-3.27l.18-.27H199l2.37,4.66H187.89l2.37-4.66h1.44Zm10.65,4.64-2.84-5.59a.47.47,0,0,0-.42-.26h-1.14a9.49,9.49,0,0,0,1.66-4.23,5.17,5.17,0,0,0-5.17-5.17,5.17,5.17,0,0,0-5.17,5.17,9.51,9.51,0,0,0,1.66,4.23H190a.47.47,0,0,0-.42.26l-2.85,5.59a.47.47,0,0,0,0,.46.47.47,0,0,0,.4.22h15a.47.47,0,0,0,.4-.22A.47.47,0,0,0,202.52-389.75Z"
    fill="#a4a1a1"
  />
  <path
    className="markerIconPath"
    d="M237.19-399.79A1.19,1.19,0,0,1,236-398.6a1.19,1.19,0,0,1-1.19-1.19A1.19,1.19,0,0,1,236-401a1.19,1.19,0,0,1,1.19,1.19ZM236-401.91a2.13,2.13,0,0,0-2,1.31,2.13,2.13,0,0,0,.46,2.32,2.13,2.13,0,0,0,2.32.46,2.13,2.13,0,0,0,1.31-2A2.13,2.13,0,0,0,236-401.91Z"
    fill="#a4a1a1"
  />
  <path
    className="markerIconPath"
    d="M254.66-398.26a43.58,43.58,0,0,1-2.63-3.81,9.21,9.21,0,0,1-1.6-3.86,4.23,4.23,0,0,1,2.11-3.66,4.23,4.23,0,0,1,4.23,0,4.23,4.23,0,0,1,2.11,3.66,9.22,9.22,0,0,1-1.6,3.86A43.59,43.59,0,0,1,254.66-398.26Zm-2.74-2.22c1.17,1.78,2.36,3.26,2.38,3.27a.47.47,0,0,0,.36.17.47.47,0,0,0,.36-.17s1.21-1.5,2.37-3.27l.18-.27H259l2.37,4.66H247.93l2.37-4.66h1.44Zm10.65,4.64-2.84-5.59a.47.47,0,0,0-.42-.26h-1.14a9.49,9.49,0,0,0,1.66-4.23,5.17,5.17,0,0,0-5.17-5.17,5.17,5.17,0,0,0-5.17,5.17,9.51,9.51,0,0,0,1.66,4.23H250a.47.47,0,0,0-.42.26l-2.85,5.59a.47.47,0,0,0,0,.46.47.47,0,0,0,.4.22h15a.47.47,0,0,0,.4-.22A.47.47,0,0,0,262.56-395.84Z"
    fill="#a4a1a1"
  />
  <path
    className="markerIconPath"
    d="M49.21,7.74A29.37,29.37,0,0,0,19.87,37.08c0,17.91,21.49,44.22,29.34,53.24,7.85-9,29.35-35.35,29.35-53.23A29.38,29.38,0,0,0,49.21,7.74ZM59.15,37.5a9.94,9.94,0,1,1-9.94-9.94A9.94,9.94,0,0,1,59.15,37.5Z"
    fill="#a4a1a1"
  />
</svg>
`;
